R/check-input-helpers.R

```diff
@@ -0,0 +1,390 @@
+#' @title Check whether an input is an atomic vector of mode 'numeric'
+#'
+#' @description Helper function
+#' @param x input to check
+#' @param x additional arguments to pass to `check_numeric()`
+#' @importFrom checkmate check_atomic_vector check_numeric
+#' @return Either TRUE if the test is successful or a string with an error
+#' message
+#' @keywords internal
+check_numeric_vector <- function(x, ...) {
```

Comment:
be consistent if you want the random spaces

Reply:
Made a note in #346 and will address that later to avoid more merge conflicts